### ISSUE TITLE
p2p daemon gives panic on stop action

### ISSUE THREAD

**Issue by soffokl** [subutai-io/p2p#160]
`/apps/subutai/current/bin/p2p stop -hash swarm-a31d106e-91f4-44c6-819f-1492bd18cc79`

In daemon logs see following: 
```
Jun 24 09:38:08 rh1466751583.gw.intra.lan ubuntu-core-launcher[3458]: panic: runtime error: invalid memory address or nil pointer dereference
Jun 24 09:38:08 rh1466751583.gw.intra.lan ubuntu-core-launcher[3458]: [signal 0xb code=0x1 addr=0x0 pc=0x4c5f18]
Jun 24 09:38:08 rh1466751583.gw.intra.lan ubuntu-core-launcher[3458]: goroutine 85 [running]:
Jun 24 09:38:08 rh1466751583.gw.intra.lan ubuntu-core-launcher[3458]: panic(0x8aa320, 0xc8200100b0)
Jun 24 09:38:08 rh1466751583.gw.intra.lan ubuntu-core-launcher[3458]: /usr/lib/go/src/runtime/panic.go:481 +0x3e6
Jun 24 09:38:08 rh1466751583.gw.intra.lan ubuntu-core-launcher[3458]: github.com/subutai-io/p2p/lib.(*PTPCloud).StopInstance(0xc820210600)
Jun 24 09:38:08 rh1466751583.gw.intra.lan ubuntu-core-launcher[3458]: /home/crioto/Projects/golang/src/github.com/subutai-io/p2p/lib/p2p.go:764 +0xab8
Jun 24 09:38:08 rh1466751583.gw.intra.lan ubuntu-core-launcher[3458]: main.(*Procedures).Stop(0xc820124430, 0xc8202923a0, 0xc8202503e0, 0x0, 0x0)
Jun 24 09:38:08 rh1466751583.gw.intra.lan ubuntu-core-launcher[3458]: /home/crioto/Projects/golang/src/github.com/subutai-io/p2p/instance.go:273 +0x253
Jun 24 09:38:08 rh1466751583.gw.intra.lan ubuntu-core-launcher[3458]: reflect.Value.call(0x86ec00, 0x900c58, 0x13, 0x93ed20, 0x4, 0xc8202b9ee8, 0x3, 0x3, 0x0, 0x0, ...)
Jun 24 09:38:08 rh1466751583.gw.intra.lan ubuntu-core-launcher[3458]: /usr/lib/go/src/reflect/value.go:435 +0x120d
Jun 24 09:38:08 rh1466751583.gw.intra.lan ubuntu-core-launcher[3458]: reflect.Value.Call(0x86ec00, 0x900c58, 0x13, 0xc8202b9ee8, 0x3, 0x3, 0x0, 0x0, 0x0)
Jun 24 09:38:08 rh1466751583.gw.intra.lan ubuntu-core-launcher[3458]: /usr/lib/go/src/reflect/value.go:303 +0xb1
Jun 24 09:38:08 rh1466751583.gw.intra.lan ubuntu-core-launcher[3458]: net/rpc.(*service).call(0xc820138780, 0xc820138240, 0xc8202a0ce0, 0xc82014a400, 0xc820450bc0, 0x7c4400, 0xc8202923a0, 0x16, 0x7c4340, 0xc8202
Jun 24 09:38:08 rh1466751583.gw.intra.lan ubuntu-core-launcher[3458]: /usr/lib/go/src/net/rpc/server.go:383 +0x1c2
Jun 24 09:38:08 rh1466751583.gw.intra.lan ubuntu-core-launcher[3458]: created by net/rpc.(*Server).ServeCodec
Jun 24 09:38:08 rh1466751583.gw.intra.lan ubuntu-core-launcher[3458]: /usr/lib/go/src/net/rpc/server.go:477 +0x49d
```

**Comment by soffokl** [comment]
Latest version(`p2p Cloud project 4.0.0-RC14-29-ga99b4d6. Packet version: 5`) of p2p gives following panic: 

```
Jun 29 12:03:01 rh1466751583.gw.intra.lan ubuntu-core-launcher[9982]: panic: runtime error: invalid memory address or nil pointer dereference
Jun 29 12:03:01 rh1466751583.gw.intra.lan ubuntu-core-launcher[9982]: [signal 0xb code=0x1 addr=0x0 pc=0x4c6cdc]
Jun 29 12:03:01 rh1466751583.gw.intra.lan ubuntu-core-launcher[9982]: goroutine 115 [running]:
Jun 29 12:03:01 rh1466751583.gw.intra.lan ubuntu-core-launcher[9982]: panic(0x8ab5a0, 0xc8200100b0)
Jun 29 12:03:01 rh1466751583.gw.intra.lan ubuntu-core-launcher[9982]: /usr/lib/go/src/runtime/panic.go:481 +0x3e6
Jun 29 12:03:01 rh1466751583.gw.intra.lan ubuntu-core-launcher[9982]: github.com/subutai-io/p2p/lib.(*PTPCloud).StopInstance(0xc8201fe780)
Jun 29 12:03:01 rh1466751583.gw.intra.lan ubuntu-core-launcher[9982]: /home/crioto/Projects/golang/src/github.com/subutai-io/p2p/lib/p2p.go:773 +0xafc
Jun 29 12:03:01 rh1466751583.gw.intra.lan ubuntu-core-launcher[9982]: main.(*Procedures).Stop(0xc820011978, 0xc82042e2b0, 0xc8204005a0, 0x0, 0x0)
Jun 29 12:03:01 rh1466751583.gw.intra.lan ubuntu-core-launcher[9982]: /home/crioto/Projects/golang/src/github.com/subutai-io/p2p/instance.go:273 +0x253
Jun 29 12:03:01 rh1466751583.gw.intra.lan ubuntu-core-launcher[9982]: reflect.Value.call(0x86fe80, 0x902418, 0x13, 0x940518, 0x4, 0xc820279ee8, 0x3, 0x3, 0x0, 0x0, ...)
Jun 29 12:03:01 rh1466751583.gw.intra.lan ubuntu-core-launcher[9982]: /usr/lib/go/src/reflect/value.go:435 +0x120d
Jun 29 12:03:01 rh1466751583.gw.intra.lan ubuntu-core-launcher[9982]: reflect.Value.Call(0x86fe80, 0x902418, 0x13, 0xc820279ee8, 0x3, 0x3, 0x0, 0x0, 0x0)
Jun 29 12:03:01 rh1466751583.gw.intra.lan ubuntu-core-launcher[9982]: /usr/lib/go/src/reflect/value.go:303 +0xb1
Jun 29 12:03:01 rh1466751583.gw.intra.lan ubuntu-core-launcher[9982]: net/rpc.(*service).call(0xc82000be00, 0xc82000b8c0, 0xc820202390, 0xc82008e580, 0xc82026e3a0, 0x7c5400, 0xc82042e2b0, 0x16, 0x7c5340, 0xc8204
Jun 29 12:03:01 rh1466751583.gw.intra.lan ubuntu-core-launcher[9982]: /usr/lib/go/src/net/rpc/server.go:383 +0x1c2
Jun 29 12:03:01 rh1466751583.gw.intra.lan ubuntu-core-launcher[9982]: created by net/rpc.(*Server).ServeCodec
Jun 29 12:03:01 rh1466751583.gw.intra.lan ubuntu-core-launcher[9982]: /usr/lib/go/src/net/rpc/server.go:477 +0x49d
```

**Comment by crioto** [comment]
Fixed in 4.0.0
